data-raw/create_adams_data.R

```diff
@@ -109,7 +134,7 @@ run_template <- function(tp) {
       # write labels
       data <- write_labels(data, dataset_name, suffix)
 
-      # save it to pharmaverseadam data package
+      # save file to pharmaverseadam data dir (pharmaverseadam/data/adae.rda)
```

Comment:
```suggestion
      # save file to pharmaverseadam data dir (pharmaverseadam/data/<adamname>.rda)
```

Reply:
I will adopt this your notation (  pharmaverseadam/data/<adamname>.rda)